src/snowflake/connector/cursor.py

```diff
@@ -576,7 +575,6 @@ def execute(
         _is_internal: bool = False,
         _describe_only: bool = False,
         _no_results: bool = False,
-        _use_ijson: bool = False,
```

Comment:
This argument was deprecated in 2020 https://docs.snowflake.com/en/release-notes/2020-05.html#snowflake-connector-for-python-remove-support-for-ijson-library